model.py

```diff
@@ -106,15 +82,21 @@ def inference(self, device):
         with torch.no_grad():
             test_y_pred = self.net(utils.test_X.float().cuda(device)).squeeze()
             test_loss = mse(utils.test_Y.float().cuda(device), test_y_pred).detach().item()
-        print("Testing Loss: " + str("%.5f" % test_loss))
+        print("Testing Loss: " + str("%.5f" % test_loss) + " MAPE: " + str(mape(utils.test_Y.float(), test_y_pred.cpu())) )
         nbins = 100
-        plt.hist(utils.test_Y.numpy(), bins=nbins, histtype = "step", color = "r", label = "Test Dataset")
-        plt.hist(test_y_pred.detach().cpu().numpy(), bins=nbins, histtype = "step", label = "DNN Prediction")
-        plt.xlabel("-log_10(DY weight)")
-        plt.ylabel("events")
-        plt.legend()
+        # plt.hist(utils.test_Y.numpy(), bins=nbins, histtype = "step", color = "r", label = "Test Dataset")
+        # plt.hist(test_y_pred.detach().cpu().numpy(), bins=nbins, histtype = "step", label = "DNN Prediction")
+        # plt.xlabel("-log_10(DY weight)")
+        # plt.ylabel("events")
+        # plt.legend()
+        # plt.savefig(self.manager.args["save_testing_histogram_at"])
```

Comment:
```suggestion
```

Probably better to remove code rather than have commended out code in version control.